js/deleteTask.js

```diff
@@ -0,0 +1,6 @@
+export default function removeTodo(element, taskList, id) {
+  const index = taskList.findIndex((item) => item.id.toString() ===id);
```

Comment:
converting every task id to `String` is more expensive operation than to converting just the current task id to Number.

when calling the `removeTodo` function, parameter `id` can be converted to Number ex. `removeTodo(element, taskList, Number(element.id))`

```suggestion
  const index = taskList.findIndex((item) => item.id === id);
```